src/test/java/core/basesyntax/FileWorkTest.java

```diff
@@ -16,15 +18,15 @@
             new String[]{"wall", "wave", "width", "world", "www"};
 
     @Test
-    public void readFromEmptyFile() {
+    public void readFromEmptyFile() throws IOException {
```

Comment:
you are not allowed to change tasks. please, create solution, which doesn't throw an Exception